libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -1031,6 +1031,21 @@
   <!-- Geometric nodes                                                          -->
   <!-- ======================================================================== -->
 
+  <nodegraph name="NG_bump_vector3" nodedef="ND_bump_vector3">
+    <heighttonormal name="N_heighttonormal" type="vector3">
+      <input name="in" type="float" interfacename="height" />
+      <input name="scale" type="float" interfacename="scale" />
+    </heighttonormal>
+    <normalmap name="N_normalmap" type="vector3">
+      <input name="in" type="vector3" nodename="N_heighttonormal" />
+      <input name="normal" type="vector3" interfacename="normal" />
+      <input name="space" type="string" value="tangent" />
+      <input name="scale" type="float" value="1" />
```

Comment:
A few small notes:
* It seems it would be useful to expose normalmap `scale` and `space` as controllable by the user.
* Also the current scale is not documented as to what it's scaling :). 
* Curious about the nodegroup as `geometric` as this has been used solely for geometric stream so far. Would `texture2d` be a possible alternative. heighttonormal is `convolution2d` and normalmap is `math` (?).

Reply:
@kwokcb  thanks, good nodes; for the last note about the group, I'd be happy to pick one, but I find it super strange from an artist standpoint to see those nodes in either of those categories. To be fair, it's not as user-facing as most tools like Houdini's tab menu are probably searched by name. But I find it strange that both are grouped in that way; texture2d also doesn't seem applicable in this case.

Reply:
@kwokcb  actually that first note, wouldn't adding a second scale be redundant? The scale input is a multiplier on the amount of input bump. We should probably plug the bump node's scale into the normalmap's scale, rather than heighttonormal.